eng/scripts/Create-APIView.ps1

```diff
@@ -166,6 +187,64 @@ function Invoke-SwaggerAPIViewParser {
     }
 }
 
+<#
+.DESCRIPTION
+  Invoke the TypeSpec parset to generate APIView tokens.
+
+.PARAMETER Type
+  New or Baseline TypeSpec APIView tokens.
+
+.PARAMETER ProjectPath
+  The TypeSpec Project path.
+
+.PARAMETER ResourceProvider
+  The ResourceProvider Name.
+
+.PARAMETER Tag
+  The Tag to use for generating the APIView Tokens.
+
+.PARAMETER TokenDirectory
+  The directory to store the generated APIView Tokens.
+
+.OUTPUTS
+  The resource provider name.
+#>
+function Invoke-TypeSpecAPIViewParser {
+  param (
+      [ValidateSet("New", "Baseline")]
+      [Parameter(Mandatory = $true)]
+      [string]$Type,
+      [Parameter(Mandatory = $true)]
+      [string]$ProjectPath,
+      [Parameter(Mandatory = $true)]
+      [string]$ResourceProvider,
+      [Parameter(Mandatory = $true)]
+      [string]$TokenDirectory
+  )
+  $tempWorkingDirectoryName = [guid]::NewGuid().ToString()
+  $tempWorkingDirectoryPath = [System.IO.Path]::Combine($TempDirectory, $tempWorkingDirectoryName)
+  New-Item -ItemType Directory -Path $tempWorkingDirectoryPath > $null
+
+  try {
+    Write-Host "Compiling files and generating '$Type' APIView for '$resourceProvider'..."
+    Push-Location $ProjectPath
+    npm exec --no -- tsp compile . --emit=@azure-tools/typespec-apiview --option @azure-tools/typespec-apiview.emitter-output-dir=$tempWorkingDirectoryPath/output/apiview.json
```

Comment:
Do you want to check the error here?

Reply:
what do you mean by this?

Reply:
I think he is asking about checking error codes from the call.